package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "primer-figma",
+  "version": "0.3.0",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lukasoppermann/figmaWebhook.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/lukasoppermann/figmaWebhook/issues"
+  },
+  "homepage": "https://github.com/lukasoppermann/figmaWebhook#readme",
```

Comment:
FYI, these should point at primer org now?

Reply:
Very good point, will fix.